server/belga/io/feed_parsers/belga_newsml_1_2.py

```diff
@@ -45,7 +45,7 @@ class BelgaNewsMLOneFeedParser(BaseBelgaNewsMLOneFeedParser):
 
     label = 'Belga News ML 1.2 Parser'
 
-    SUPPORTED_TEXT_ASSET_TYPES = ('ALERT', 'SHORT', 'TEXT', 'BRIEF', 'ORIGINAL')
+    SUPPORTED_TEXT_ASSET_TYPES = ('ALERT', 'SHORT', 'TEXT', 'BRIEF', 'ORIGINAL', 'QUOTE')
     SUPPORTED_MEDIA_ASSET_TYPES = {
```

Comment:
I would expect the QUOTE rather in the AUDIO section of the SUPPORTED_MEDIA_ASSET_TYPES, that wouldn't work?

Reply:
I've tried before, but it didn't work.

Reply:
ok but we should rather fix it, it's confusing to have audio type together with those text ones

Reply:
So you mean we have to remove **audio** from the `SUPPORTED_MEDIA_ASSET_TYPES` ?

Reply:
I mean we should remove QUOTE from SUPPORTED_TEXT_ASSET_TYPES and have it in the AUDIO section,
because it's not text we're ingesting there